docs/angular-integration.asciidoc

```diff
@@ -15,43 +15,42 @@ Install the `@elastic/apm-rum-angular` package as a dependency to your applicati
 
 [source,bash]
 ----
-npm install @elastic/apm-rum-angular --save
+npm install @elastic/apm-rum-angular@alpha --save
 ----
 
+NOTE: If you are using an Angular version < 9.0, use @elastic/apm-rum-angular 1.x to instrument your application. Details are available in a https://www.elastic.co/guide/en/apm/agent/rum-js/4.x/angular-integration.html[prior release].
+
 [float]
 ==== Instrumenting your Angular application
 
-The Angular integration package comes with a `ApmService` which uses Angular Depedency injection pattern and
-will start subscribing to https://angular.io/api/router/Event[Angular Router Events] once the service is initialized.
+The Angular integration packages exposes the `ApmModule` and `ApmService` which uses Angular's dependency injection
+pattern and will start subscribing to https://angular.io/api/router/Event[Angular Router Events] once the service is initialized.
 
 `ApmService` must be initialized from either the application module or application component since
 the RUM agent has to start capturing all the resources and API calls as soon as possible.
 
 
 [source,js]
 ----
-import { NgModule, Inject } from '@angular/core'
-import { Routes, Router, RouterModule } from '@angular/router'
-import { ApmService } from '@elastic/apm-rum-angular'
+import { NgModule } from '@angular/core'
+import { BrowserModule } from '@angular/platform-browser'
+import { Routes, RouterModule } from '@angular/router'
+import { ApmModule, ApmService } from '@elastic/apm-rum-angular'
 
 const routes: Routes = [
   { path: 'contact', component: ContactListComponent },
   { path: 'contact/:id', component: ContactDetailComponent }
 ]
 
 @NgModule({
-  imports: [BrowserModule, RouterModule.forRoot(routes)],
+  imports: [ApmModule, BrowserModule, RouterModule.forRoot(routes)],
   declarations: [AppComponent, ContactListComponent, ContactDetailComponent],
-  providers: [{
-    provide: ApmService,
-    useClass: ApmService,
-    deps: [Router]
-  }],
+  providers: [ApmService],
```

Comment:
Specifying the `ApmService` in local providers shouldn't be necessary as it's already `providedIn: 'root'`

Reply:
Neat, Thanks for the tip :)